### PR TITLE
git-mv: update page

### PR DESCRIPTION
**Pull request by kbdharun**
From https://github.com/tldr-pages/tldr/pull/9578

### CODE REVIEW

File: pages/common/git-mv.md [hunk 1]
```diff
@@ -7,10 +7,10 @@
 
 `git mv {{path/to/file}} {{new/path/to/file}}`
 
-- Rename file and add renaming to the next commit:
+- Rename file or directory and add renaming to the next commit:
```
Comment (acuteenvy) [inline pages/common/git-mv.md:10]:
```suggestion
- Rename a file or directory and add the renaming to the next commit:
```

**Review by acuteenvy — COMMENTED** [review]
The first example is also missing an "a", but I couldn't make a suggestion for it.
> Move **a** file inside the repo and add the movement to the next commit:

**Review by acuteenvy — APPROVED** [review]

**Review by pixelcmtd — COMMENTED** [review]
aren't example 1 and 2 the same? could we collapse them into one?

**Comment by kbdharun** [comment]
> aren't example 1 and 2 the same? could we collapse them into one?

Even though the command is same for both the examples, I think the context and placeholders for both the cases differ so I think it is fine as the page only has 3 examples.

File: pages/common/git-mv.md [hunk 2]
```diff
@@ -3,14 +3,14 @@
...
-- Rename file and add renaming to the next commit:
+- Rename a file or directory and add the renaming to the next commit:
 
-`git mv {{filename}} {{new_filename}}`
+`git mv {{path/to/file_or_directory}} {{path/to/destination}}`
 
 - Overwrite the file in the target path if it exists:
 
-`git mv --force {{file}} {{target}}`
+`git mv --force {{file}} {{path/to/target}}`
```
Comment (pixelcmtd) [inline pages/common/git-mv.md:16]:
```suggestion
`git mv --force {{path/to/file_or_directory}} {{path/to/destination}}`
```

File: pages/common/git-mv.md [hunk 3]
```diff
@@ -3,14 +3,14 @@
...
 
 `git mv {{path/to/file}} {{new/path/to/file}}`
 
-- Rename file and add renaming to the next commit:
+- Rename a file or directory and add the renaming to the next commit:
 
-`git mv {{filename}} {{new_filename}}`
+`git mv {{path/to/file_or_directory}} {{path/to/destination}}`
 
 - Overwrite the file in the target path if it exists:
```
Comment (pixelcmtd) [inline pages/common/git-mv.md:14]:
```suggestion
- Overwrite the file or directory in the target path if it exists:
```

**Comment by pixelcmtd** [comment]
> Even though the command is same for both the examples, I think the context and placeholders for both the cases differ so I think it is fine as the page only has 3 examples.

Well, one example talks about moving files, the other talks about renaming them. I really don't see why we should keep it, but whatever idc

**Pull request merged**